.github/workflows/ci.yml

```diff
@@ -123,6 +161,10 @@ jobs:
           poetry run alembic downgrade -1
           poetry run alembic upgrade head
 
+      - name: Generate required files
+        run: |
```

Comment:
The "|" is not needed right? as this is not a multi line command

Reply:
It also does not hurt. keeping it